index.js

```diff
@@ -19,3 +21,155 @@ links.forEach((n) => n.addEventListener('click', () => {
   nav.classList.remove('show');
   menuToggle.style.display = 'block';
 }));
+
+const project = [
+  {
+    src: 'images/61.png',
+    title: 'Tonic',
+    description: 'A daily selection of privately personalized reads no accounts or sign-ups required',
+  },
+  {
+    src: 'images/4.png',
+    title: 'Multi-Post Stories',
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+  },
+  {
+    src: 'images/1.png',
+    title: 'Facebook 360',
+    description: 'Exploring the future of media in Facebook first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+  },
+  {
+    src: 'images/2.png',
+    title: 'Uber Navigation',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car. Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+  },
+];
```

Comment:
- [ ] I can see that you have the `project` array of objects here and below there is `popups` array of the same project objects which is kinda repeating and I have seen that you are not using this. Please remove it 😃  For respecting the DRY principle